packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -81,6 +86,10 @@ class ExpansionPanel {
   /// The widget builder that builds the expansion panels' header.
   final ExpansionPanelHeaderBuilder headerBuilder;
 
+  /// The widget builder that builds the expansion panels' indicator.
+  /// If null, use [ExpandIcon] for default
```

Comment:
Newline between these two lines. 

Public-facing docs should always have only one sentence as the first line, then followed by a newline and the rest of the documentation.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements

Reply:
got it.  Updating this code ...